s4e-web/cypress/integration/map/map-data-search.spec.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference types="Cypress" />
+
+import { Login } from '../../page-objects/auth/auth-login.po';
+import { MapDataSearch } from '../../page-objects/map/map-data-search.po';
+
+before(function () {
+  cy.fixture('users/zkMember.json').as('zkMember');
+});
+
+describe.only('Map Data Search', () => {
+
+  context("Data search for not logged in user", () => {
+
+    before(() => {
+      cy.visit('/login');
+    });
+
+    it('shouldn\'t download data without authentication', () => {
+      Login
+        .goToMapWithoutLogin()
```

Comment:
Why you haven't use just cy.visit('/map')? In such case think about KISS (Keep It Simple, Silly).

Reply:
I created this method for other test where I check the button functionality and I reuse in few places. This method also wait for suitable path where products are loaded, so I think reuse this method is more consistent and have sense.

Reply:
In that case please rename it just to `goToMap`.